eclipse-templatevariables-feature/feature.xml

```diff
@@ -194,34 +194,39 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
+      boilerplate notice, with the fields enclosed by brackets &quot;[]&quot;
+      replaced with your own identifying information. (Don&apos;t include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
+      same &quot;printed page&quot; as the copyright notice for easier
       identification within third-party archives.
 
    Copyright [yyyy] [name of copyright owner]
 
-   Licensed under the Apache License, Version 2.0 (the "License");
+   Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
 
        http://www.apache.org/licenses/LICENSE-2.0
 
    Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
+   distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.      
+   limitations under the License.
    </license>
 
    <url>
       <update label="TemplateVariables Update Site" url="http://m-m-m.github.io/eclipse-templatevariables/"/>
       <discovery label="mmm TempalteVariables" url="http://m-m-m.github.io/eclipse-templatevariables/"/>
    </url>
 
-   <plugin id="net.sf.mmm.eclipse.templatevariables.plugin" version="0.0.0" unpack="false"/>
+   <plugin
+         id="net.sf.mmm.eclipse.templatevariables.plugin"
+         download-size="0"
+         install-size="0"
+         version="1.1.0.qualifier"
```

Comment:
fine. So `0.0.0` was more a hack.
A perfect fix would be to use `${project.version}.qualifier` in combination with maven filtering.

Reply:
You do not need to do this change. I can take it over. You already improved/fixed this versioning and it is fine for the upcoming release `1.1.0` so no need to change for this PR.